src/App.vue

```diff
@@ -2,7 +2,12 @@
 import { useSettings } from '@stores/useSettings'
 import ModalBox from '@components/ModalBox.vue'
 import { updateAuthKey } from '@/api'
+import useStoreMapping from '@/utils/storeRouteMapping'
+import vLock from '@/directives/vLock'
```

Comment:
Why did you import the directive? You already declared it globally in the `main.ts` file